api_app/api/routes/airlock_resource_helpers.py

```diff
@@ -140,6 +142,35 @@ def enrich_requests_with_allowed_actions(requests: List[AirlockRequest], user: U
     return enriched_requests
 
 
+async def delete_review_resource(
```

Comment:
Nit: maybe name it `delete_one_review_user_resource` for parity with `delete_review_user_resources`?

Reply:
I'll rename it to `delete_review_user_resource`, then rename the other method to `delete_all_review_user_resources` to make it clear